### PR TITLE
Providing a way to set log level for messages coming from SDK

### PR DESCRIPTION
**Pull request by wmathurin**
Default log level is info so by default it will be a lot less verbose than before.
To change the log level simply do:

```javascript
import { forceLog } from 'react-native-force';
forceLog.setLogLevel("debug"); // other valid values: info, warn, error
```

### CODE REVIEW

File: src/react.force.log.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+export const setLogLevel = (level) => {
+    logLevel = level;
+    var methods = ["debug", "info", "warn", "error"];
+    var levelAsInt = methods.indexOf(level.toLowerCase());
+    var noop = () => {};
+
+    sdkConsole.debug = levelAsInt <= 0 ? console.debug.bind(console) : noop;
+    sdkConsole.info = levelAsInt <= 1 ? console.info.bind(console) : noop;
+    sdkConsole.warn = levelAsInt <= 2 ? console.log.bind(console) : noop; // we don't want the yellow box
```
Comment (bbirman) [inline src/react.force.log.js:47]: Why does warn map to log?

Comment (wmathurin) [reply src/react.force.log.js:47]: When you do console.warn in a react native app, you end up with a big yellow warning on the screen. When you do console.error in a react native app, it ends up in a big red box on the screen. I didn't want that behavior for our errors/warnings but what do you think?

Comment (bbirman) [reply src/react.force.log.js:47]: Sounds good to me! Is line 48 supposed have log too then? (sorry if I'm misunderstanding)

Comment (wmathurin) [reply src/react.force.log.js:47]: Good catch.

**Pull request merged**